README.md

```diff
@@ -32,7 +32,7 @@
 [![JetBrains Plugins](https://img.shields.io/jetbrains/plugin/v/14221-git-machete.svg)](https://plugins.jetbrains.com/plugin/14221-git-machete)
 [![Downloads](https://img.shields.io/jetbrains/plugin/d/14221-git-machete.svg)](https://plugins.jetbrains.com/plugin/14221-git-machete)
 
-<img src="src/main/resources/META-INF/pluginIcon.svg" width="130" height="130" align="left"/>
+<img src="docs/logo.svg" style="width: 100%; display: block; margin-bottom: 10pt;"/>
```

Comment:
Hmm no chance for the original git font? if it's paid or sth, I can sort it out :)

Reply:
TBH I have mixed feelings about have black&red text (I'd say keep it black)... WDYT @mkondratek ?

Reply:
`Paint it, black` :black_circle: we can experiment easily (but I like this first version)

actually it is a very dark brown, right? 

@micpiotrowski can you please provide here SSs with this one and other versions? ideally with some context like below

![image](https://user-images.githubusercontent.com/19799111/90824494-ce2d4080-e337-11ea-8430-5e431fd1958a.png)

Reply:
the last one wins! :trophy: